### PR TITLE
Conflict solved

### PR DESCRIPTION
**Pull request by prajkta21**
check the code

### CODE REVIEW

File: src/java/util/Hashmap.java [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+	hp.put(4, "Prajkta");
+
 	hp.put(1, "Deepa");
 	hp.put(2, "snehal");
 	hp.put(3, "Mahesh");
->>>>>>> af8e64a5d68762f0d855956f3ffed4fb71b5c6a7
+
 	System.out.println(hp);
 }
 }
```
Comment (snehaldorlikar) [inline src/java/util/Hashmap.java:17]: clean the code

**Review by snehaldorlikar — CHANGES_REQUESTED** [review]

File: src/java/util/Hashmap.java (hunk 1 above)
Comment (prajkta21) [reply src/java/util/Hashmap.java:17]: Already clean

**Review by snehaldorlikar — APPROVED** [review]
okay

**Comment by snehaldorlikar** [comment]
confirm merge

**Pull request merged**